### PR TITLE
Cleanup U2F namespace to correspond to folder path

### PR DESCRIPTION
**Pull request by rockstardev**

### CODE REVIEW

File: BTCPayServer/BTCPayServer.csproj [hunk 1]
```diff
@@ -6,18 +6,22 @@
...
     <Compile Remove="wwwroot\bundles\jqueryvalidate\**" />
     <Compile Remove="wwwroot\css\**" />
     <Compile Remove="wwwroot\vendor\jquery-nice-select\**" />
     <Content Remove="Build\dockerfiles\**" />
+    <Content Remove="U2F\Services\**" />
     <Content Remove="wwwroot\bundles\jqueryvalidate\**" />
     <Content Remove="wwwroot\css\**" />
     <Content Remove="wwwroot\vendor\jquery-nice-select\**" />
     <EmbeddedResource Remove="Build\dockerfiles\**" />
+    <EmbeddedResource Remove="U2F\Services\**" />
```
Comment (Kukks) [inline BTCPayServer/BTCPayServer.csproj]: I wonder why this is an embedded resource?

Comment (rockstardev) [reply BTCPayServer/BTCPayServer.csproj]: We have a lot of those in csproj thanks to IDE bugs... I'll clean it up in next few days.

Comment (Kukks) [reply BTCPayServer/BTCPayServer.csproj]: That's fine as long as they dont do anything deadly.

Comment (rockstardev) [reply BTCPayServer/BTCPayServer.csproj]: https://github.com/btcpayserver/btcpayserver/pull/1068/commits/755a6bf8e6c35cda1fdc4d8adecca4ba21b4d157

**Comment by NicolasDorier** [comment]
tests do not pass?

**Pull request merged**